bloom/commands/git/import_upstream.py

```diff
@@ -205,6 +206,12 @@ def handle_tree(tree, directory, root_path, version):
                 file_data = show(BLOOM_CONFIG_BRANCH, os.path.join(root_path, rel_path))
             # Write file
             with open(rel_path, 'wb') as f:
+                # Python 2 will treat this as an ascii string but
+                # Python 3 will not re-decode a utf-8 string.
+                try:
+                    file_data = file_data.encode('utf-8')
+                except UnicodeDecodeError:
+                    file_data = file_data.decode('utf-8').encode('utf-8')
```

Comment:
> Does this actually do anything? I guess it will still throw on a invalid character for decoding as utf-8. Would it be better to do an if python3 then ... rather than doing a try-except?

Sadly there's enough going on in the UI here that I'm not sure which line this is questioning. Can you drop a line number or quote some code? Thanks!